wagtail/admin/views/tags.py

```diff
@@ -19,7 +19,7 @@ def autocomplete(request, app_name=None, model_name=None):
 
     term = request.GET.get("term", None)
     if term:
-        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")
+        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")[:10]
```

Comment:
Can you please also add Python tests for this.

I would suggest we include a comment as to why we have filtered these here.

```suggestion
        # limit the tags provided to the client for performance when projects have a large amount of tags
        tags = tag_model.objects.filter(name__istartswith=term).order_by("name")[:10]
```

Reply:
Yes, I'll look into python testing for this. Thanks